extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-validation.service.ts

```diff
@@ -109,8 +106,8 @@ export class DragAndDropSortInputValidationService {
       var noOfMismatches = 0;
       var inputs = earlierRule.inputs.x;
       var answer = laterRule.inputs.x;
-      for (var i = 0; i < math.min(inputs.length, answer.length); i++) {
-        for (var j = 0; j < math.max(answer[i].length, inputs[i].length);
+      for (var i = 0; i < Math.min(inputs.length, answer.length); i++) {
```

Comment:
@iamprayush One question on this -- have you checked the min/max methods in the math library and verified that they are functionally equivalent to the standard Math.min and Math.max? It's always worth verifying this sort of thing, to prevent any differences in functionality arising due to the refactor.

Reply:
@seanlip The difference between the two methods according to the documentation is that the mathjs min/max functions also allow minimum/maximum of matrices based on a given dimension. But for normal integer inputs, both functions work the same. Seeing that in this case only integer inputs are specified, the JS min/max functions would work the same.

Reply:
Sounds good -- I have no further concerns. Thanks for checking!